### ISSUE TITLE
single-member multi-line expressions and trailing commas

### ISSUE THREAD

**Issue by mattsb42-aws** [psf/black#493]
Operating system: MacOS
Python version: 3.7.0
*Black* version: 18.6b4
Does also happen on master: yes

I can't quite tell if this is intended behavior or if it is emergent behavior caused by other things.

I noticed this because `black` and `isort` (configured with `multi_line_output = 3` and `include_trailing_comma = True`) started fighting over a particular line.

Per the trailing comma rules, it it my understanding that all multi-member expressions defined over multiple lines should have a trailing comma at the end of each member line, but expressions that fit on a single line and do not syntactically require a trailing comma will have that trailing comma removed.

This is good, and in my experience this is what `black` does. However, there is an edge case that is not directly addressed in the rules in the readme: an expression that can contain multiple members but only contains a single member and cannot fit on a single line.

Based on my understanding of the reason behind the adding trailing commas to the end of multi-line expression members (making diffs less bad), I would think that the single member in these cases should still have a trailing comma.

However, based on the "remove trailing commas on single line expressions unless syntactically necessary" exception, the trailing comma should not be present.

The behavior that `black` applies follows the rules that would be followed if the expressions were contained on a single line. Is this intentional? If so, could a mention of this exception be added to the readme?

input:
```python
from long_namespace_that_makes_the_line_too_line import (
    long_variable_name,
)


long_namespace_that_makes_the_line_too_line = (
    long_variable_name,
)


long_namespace_that_makes_the_line_too_line = [
    long_variable_name,
]


long_namespace_that_makes_the_line_too_line = {
    "foo": long_variable_name,
}


long_namespace_that_makes_the_line_too_line = {
    long_variable_name,
}
```

output:
```python
from long_namespace_that_makes_the_line_too_line import (
    long_variable_name
)


long_namespace_that_makes_the_line_too_line = (
    long_variable_name,
)


long_namespace_that_makes_the_line_too_line = [
    long_variable_name
]


long_namespace_that_makes_the_line_too_line = {
    "foo": long_variable_name
}


long_namespace_that_makes_the_line_too_line = {
    long_variable_name
}
```

**Comment by zsol** [comment]
I think we should document what black does currently, with the caveat that after #250 is fixed, your import example will also have a trailing comma.

**Comment by ambv** [comment]
This will be addressed by https://github.com/python/black/pull/826.

**Comment by tucked** [comment]
I think this is still reproducible with 19.10b0 (which appears to include #826).

**Comment by ambv** [comment]
Edge cases not covered by #826 will be solved as part of #1288.
